hubble/shell.py

```diff
@@ -177,18 +177,23 @@ def getEnvironments(args, choice, config):
     return results
 
 
-def toDict(string):
+def toDict(buf):
     """ Parse a string of 'key=value' into a dict({'key': 'value'}) """
     try:
-        if len(string) == 0:
-            print("-- Warning: executable specifed by 'opt-cmd' did not return"
+        # Convert the bytes to string
+        if six.PY3:
+            buf = str(buf.rstrip(), "UTF-8")
+        else:
+            buf = str(buf.rstrip())
```

Comment:
`buf.decode('utf-8')` will work on the python2 `str` type. No need for the `if six.PY3` thing.